### PR TITLE
fix(date-time): align props date time and introduce default-value

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Removed the slots for label/helper and created props for them instead, also introduces the prop
default-value to allow the user to select a default time to be displayed.

**Solving issue**  
Fixes: [AB#2837](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2837)

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Date-time
3. Check that the controls regarding label/helper works as inteded.
4. Try out the new custom/now value for default-value.

**Screenshots**  
-

**Additional context**  
-

### CODE REVIEW

File: tegel/src/components/datetime/datetime.tsx [hunk 1]
```diff
@@ -65,6 +133,7 @@ export class Datetime {
   // Change event isn't a composed:true by default in for input
   handleChange(e): void {
     this.customChange.emit(e);
+    console.log(this.value);
```
Comment (mJarsater) [inline tegel/src/components/datetime/datetime.tsx:136]:
```suggestion
```

**Comment by mJarsater** [comment]
Question: Is it better to use the value prop to set the default value?

**Comment by theJohnnyMe** [comment]
Props format and defaultValue - are they dependent on each other and if how?
**format:** prop looks like it is not used - but maybe you wanted to discuss it more in the group? 
**defaultValue:** Do we need to specify the format that is expected in the comment of the prop so users know in which format they need to pass the value?

**Review by theJohnnyMe — APPROVED** [review]
Nice work! LGTM!

**Pull request merged**